### ISSUE TITLE
NVDA + Firefox does not read the aria-labelledby for checkboxes when reading all or reading next item.

### ISSUE THREAD

**Issue by cochfarf** [nvaccess/nvda#9164]
### Steps to reproduce:

1. With NVDA and Firefox running, go to my Code pen
2. Use the down arrow to read the checkbox line. The aria-labelledby is not spoken, just the <label>
### Actual behavior:
aria-labelledby is not read in reading mode.
Note: If TAB is used to navigate to the checkbox the aria-labelledby is spoken correctly
### Expected behavior:
The aria-labelledby should be read in reading all or read next item.
Note: It works fine with NVDA / Chrome or JAWS Chrome.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2018.4
#### Windows version:
Windows 10 Enterpris version 10,0.17134 Build 17134
#### Name and version of other software in use when reproducing the issue:
Firefox 64.0.2
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by ehollig** [comment]
@cochfarf can you provide a link to your code pen?

**Comment by ehollig** [comment]
@cochfarf we still require your Codepen in order to replicate this issue.

**Comment by ehollig** [comment]
Closing as abandoned.
